multilep/interface/GenAnalyzer.h

```diff
@@ -13,6 +13,7 @@ class GenAnalyzer {
   private:
     static const unsigned gen_nL_max = 20;
     static const unsigned gen_nPh_max = 10;
+    static const unsigned gen_n_max = 10000;
```

Comment:
10000 seems a lot, though I guess it's ok given that you implemented the storeGenParticles boolean, so this will only be stored for a few specific samples

Reply:
I will reduce the size of these arrays. In principle, the use of vectors could look be efficient than arrays. And no need to trace the max size.

Reply:
I'm not sure you want to use vectors given the fact that you will write out the array to root files. Anyway you can specify the size that is written for each event so it is just a concern of RAM memory while running.

Reply:
Yes, but one can simply write out vectors instead of arrays.